### ISSUE TITLE
Investigate learning rate schedulers for training

### ISSUE THREAD

**Issue by jackapbutler** [OpenBioML/chemnlp#174]
Look into [default learning rate scheduler](https://huggingface.co/docs/transformers/v4.27.2/en/main_classes/optimizer_schedules#transformers.get_linear_schedule_with_warmup) provided by the Hugging Face `Trainer` and decide if we want to use it.
* Suggestions of alternatives from the literature would be welcome 
- [x] #224

**Comment by bethanyconnolly** [comment]
Chemrxiv is not helping training in its current state (chemistry performance stays random but lambada performance goes to 0). This task has been down prioritised until we have a better initial dataset to train on.

**Comment by jackapbutler** [comment]
Also worth noting during multi-node training on `N` nodes we will take `steps / N` optimiser steps. We should factor in some recommendations for how this is handled if we look at lr schedulers.

**Comment by jackapbutler** [comment]
this is complete
